src/index.ts

```diff
@@ -76,6 +76,13 @@ export async function getOrientation (
   return info;
 }
 
+export async function updateOrientation (
```

Comment:
Can you delete this `updateOrientation()` function? Let's have only `updateOrientationCode()`.

The original idea of the difference between `getOrientation()` and `readOrientationCode()` is which type they return. The "code" one returns `OrientationCode` and the other returns `IOrientationInfo`. So this `updateOrientation()` should accept "info" if we implement.

I'm sorry I should've told you this before but I needed time to remember.

Reply:
This makes sense. No need to be sorry, I was asking for feedback :). I deleted the function as you requested.